### ISSUE TITLE
on ng build --prod here i got this issue

### ISSUE THREAD

**Issue by chandantech0** [highcharts/highcharts-angular#225]
ERROR in Type HighchartsChartComponent in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts is part of the declarations of 2 modules: WebsiteModule in /Users/chandan/Desktop/projet z/src/app/website.module.ts and HighchartsChartModule in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts! Please consider moving HighchartsChartComponent in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts to a higher module that imports WebsiteModule in /Users/chandan/Desktop/projet z/src/app/website.module.ts and HighchartsChartModule in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts. You can also create a new NgModule that exports and includes HighchartsChartComponent in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts then import that NgModule in WebsiteModule in /Users/chandan/Desktop/projet z/src/app/website.module.ts and HighchartsChartModule in /Users/chandan/Desktop/projet z/node_modules/highcharts-angular/highcharts-angular.d.ts.

**Comment by mateuszkornecki** [comment]
@chandantech0 
Thanks for reporting the issue! 

I was trying to reproduce the issue using the `highcharts-angular 2.8.0` and `Angular: 10.0.14` and I was able to build the project without any problem. Could you reproduce the issue in a simple demo that I could work on? Also what version of the wrapper + angular you are currently using?

**Comment by hakimio** [comment]
I am also able to build production version of the app without any issues with Angular 10. Most likely something is wrong on your end. Maybe you are importing `HighchartsChartModule` in the wrong module.

**Comment by barunantony** [comment]
Incase of ModuleNotFound Error

I thought following comment may not be helpful to resolve above issue but might help . When importing a module as shown to Readme (like below) causes ModuleNotFound error during prod build. So the call "HC_stock(Highcarts)" must be inside constructor.

-----------
import * as Highcharts from 'highcharts';
import HC_stock from 'highcharts/modules/stock';
HC_stock(Highcharts);

**Comment by hakimio** [comment]
@barunantony it's in the [README](https://github.com/highcharts/highcharts-angular#core) and the [demo app](https://github.com/highcharts/highcharts-angular/blob/master/src/app/map-chart/map-chart.component.ts#L7).

**Comment by barunantony** [comment]
Hi Tomas Rimkus @hakimio,

In the demoapp and README the code
HC_map(Highcharts);

 is written outside the Angular Class and somehow the module, HC_map is not
found on doing "ng build --prod" or "ng serve --prod". This was resolved
for me when I moved the code inside constructor (or may be it will work if
I moved inside the class) of the class.
And this was caused in Angular 7, "highcharts": "8.2.2",
"highcharts-angular": "2.8.0".

Regards,
Arun
